rgdps/logger.py

```diff
@@ -1,31 +1,147 @@
 from __future__ import annotations
 
-import logging
+import logging.config
+import sys
+import threading
+from types import TracebackType
+from typing import Any
+from typing import Callable
+from typing import Optional
 
-import structlog
 
-logger = structlog.get_logger()
+# TODO: Look into more customisability.
+_LOGGING_CONFIG = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "logzioFormat": {
+            "format": '{"additional_field": "value"}',
+            "validate": False,
+        },
+    },
+    "handlers": {
+        "logzio": {
+            "class": "logzio.handler.LogzioHandler",
+            "level": "DEBUG",
+            "formatter": "logzioFormat",
+            "token": "",
+            "logzio_type": "python",
+            "logs_drain_timeout": 5,
+            "url": "https://listener.logz.io:8071",
+        },
+    },
+    "loggers": {
+        "rgdps": {
+            "level": "DEBUG",
+            "handlers": ["logzio"],
+            "propagate": True,
+        },
+    },
+}
```

Comment:
for the akatsuki prod services i've opted to have a [logging.yaml.example](https://github.com/osuAkatsuki/bancho-service/blob/main/logging.yaml.example) in the repo and have `logging.yaml` in `.gitignore`. this also allows us to customize it dynamically (e.g. different environments might have different logging configs)

Reply:
I will investigate this in the future as the deployment practises are made more concrete (likely with #31)